src/modules/Hosts/Hosts/Strings/en-us/Resources.resw

```diff
@@ -273,16 +273,13 @@
     <value>Show only duplicates</value>
   </data>
   <data name="TooManyHostsTeachingTip.Subtitle" xml:space="preserve">
-    <value>Only 9 hosts per entry are supported</value>
+    <value>Only 9 hosts per entry are supported. The affected entries have been split. This will take effect on next change.</value>
```

Comment:
```suggestion
    <value>Only 9 hosts per entry are supported. The affected entries have been split. This will take effect on the next change.</value>
```

What does 'next change' mean, btw :)?

Reply:
No idea. I just turned three strings into two.

Reply:
This means any change from the app to the hosts file.